README.md

```diff
@@ -9,3 +10,27 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof
+
+**Considering two graphs, A and B, with the same number of nodes and are completely connected:**
+
+Graph A: $G_1 = (V_1, E_1)$
+
+Graph B: $G_2 = (V_2, E_2)$
+
+**Bijection function $f: V_1 \rightarrow V_2$, makes Graph A isomorphic to Graph B.**
```

Comment:
How do I get this function? You're assuming that it exists and has the properties you want.